docs/usage/configuration-options.md

```diff
@@ -2181,6 +2181,16 @@ Defaults to `update`, but can also be set to `branch`.
 This sets the level the postUpgradeTask runs on, if set to `update` the postUpgradeTask will be executed for every dependency on the branch.
 If set to `branch` the postUpgradeTask is executed for the whole branch.
 
+## prAllowMaintainerEdits
+
+Use `prAllowMaintainerEdits` to control if maintainers can edit Renovate's pull requests.
+
+When Renovate runs in a fork, this allows project maintainers to make manual changes to the Renovate PR branch, without needing to create another new PR.
+
+<!-- prettier-ignore -->
+!!! note
+    This option is only relevant if you set `forkToken`.
```

Comment:
```suggestion
## forkModeAllowMaintainerEdits

Use `forkModeAllowMaintainerEdits` to control if maintainers can edit Renovate's pull requests when in fork mode.

When Renovate runs in a fork, this allows project maintainers to make manual changes to the Renovate PR branch, without needing to create another new PR.

<!-- prettier-ignore -->
!!! note
    This option is only relevant if you set `forkToken`.
```


This will require reordering within the file too